src/content/docs/infrastructure/host-integrations/host-integrations-list/lighttpd-integration.mdx

```diff
@@ -112,64 +116,68 @@ integrations:
 ```
 
 </Step>
+  
+<Step>
+  
+### Forward Lighttpd logs to New Relic
+ 
+You can use our [log forwarding](https://docs.newrelic.com/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/) to forward Lighttpd logs to New Relic.
+
+On Linux machines, your log file named `logging.yml` should be present in the below path. If you don't see your log file, then you will need to create it by following the above log forwarding documentation.
+```
+Path: /etc/newrelic-infra/logging.d/
+```
+
+After creating the log file, add the below script to the `logging.yml` file:
+  
+<img width="330" alt="image" src="https://github.com/newrelic/docs-website/assets/101187392/9cabe7e7-4d92-4e82-8602-d8e94f2efe28">
+</Step  
 
 <Step>
 
-### Restart the New Relic infra agent and lighttpd service
+### Restart the infrastructure agent and lighttpd service
 
-To restart the New Relic infra agent follow [these](/docs/infrastructure/install-infrastructure-agent/manage-your-agent/start-stop-restart-infrastructure-agent/) instructions and run this command.
+Before you can start reading your data, use the instructions in our [infrastructure agent docs](https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/manage-your-agent/start-stop-restart-infrastructure-agent/) to restart your infrastructure agent.
 
 ```
 sudo systemctl restart lighttpd.service && sudo systemctl restart newrelic-infra.service
 ```
 </Step>
 
 <Step>
-###  Query your data using NRQL
+  
+###  Monitor your application
 
-You can use this sample query to view your Lighttpd metrics captured on [one.newrelic.com](https://one.newrelic.com/all-capabilities):
+You can choose our pre-built dashboard template named `Lighttpd` to monitor your Lighttpd server metrics.
+  
+Steps to use our pre-built dashboard template
+1.  Click on Add data
+2.  Click on the Dashboards tab
+3.  In the search box, type lighttpd
+4.  You should see our pre-build dashboard
+5.  Click on it to install it on your account
+
+Once your application is integrated by following the above steps, the dashboard should reflect metrics on it.  
+  
+To instrument the lighttpd quickstart and to see metrics and alerts, you can also follow our [Lighttpd quickstart page](https://newrelic.com/instant-observability/lighttpd) by clicking on the `Install now` button.
 
+Below is an example query to view the captured metrics on New Relic’s Query Builder
 ```
 FROM lighttpdStatusSample SELECT * LIMIT MAX
 ```
-```
-FROM lighttpdStatusSample2 SELECT * LIMIT MAX
-```
-
-Here, ‘lighttpdStatusSample’ and ‘lighttpdStatusSample2’ should match with the ‘event_type’ which you have mentioned in the nri-flex config file.
   
-Here's an example of NRQL query checking the server status.
+Example Query:
+Below is the query to check the active connection per server.
```

Comment:
As per the below query, we get "BusyServers an output. Hence this text has to be changed
Suggestion:
Below is the query to check the busy servers.

Reply:
It is correct. The definition of busy servers is the number of active connection per server. Still let me know if I need to do text change.

Reply:
Great.. Thank you for the insights. You can please proceed with your changes itself. 👍